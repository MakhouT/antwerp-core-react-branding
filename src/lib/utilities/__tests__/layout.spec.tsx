import * as React from "react";

import * as A from "../..";
import { renderAndMatchSnapshot } from "../../../__tests__/__fixtures__/testUtils";

describe("Layout", () => {
    it("Renders default", () => {
        // Arrange
        const element = <A.Layout
            type={A.LayoutStyle.Container}
        />;

        // Act and assert
        renderAndMatchSnapshot(element);
    });
});
