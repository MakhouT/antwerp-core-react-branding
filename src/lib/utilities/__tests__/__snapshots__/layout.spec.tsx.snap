// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout Renders default 1`] = `
<div
  className="u-container"
  style={undefined}
/>
`;
